client/src/components/Grid/GridElements/GridText.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+interface Props {
+    text?: string;
+}
+defineProps<Props>();
+</script>
+
+<template>
+    <span>{{ text ?? "Not available." }}</span>
+</template>
```

Comment:
I do not think this component is warranted

Reply:
I'd rather keep this component for now. I intend to add additional features, like a `more` button if the text is too long, and it allows for a cleaner separation between the generic grid component and cell content.